Aspekt/IgnoreWarningAttribute.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Aspekt
+{
+    [AttributeUsage(AttributeTargets.Method)]
```

Comment:
I have some thoughts about an alternative to IgnoreWarningAttribute that's a bit more targeted, but we can implement that in a separate PR.

Reply:
Yeah -- I needed something, this is similar to PostSharp. It works for now. Open to ideas.